corva/models/scheduled.py

```diff
@@ -42,13 +37,17 @@ class ScheduledEventData(BaseEventData):
     day_shift_start: Optional[str] = None
 
 
-class ScheduledEvent(ListEvent[ScheduledEventData]):
+class ScheduledEvent(BaseEvent, ScheduledEventData):
```

Comment:
Why do we have scheduled event updates in this PR?

Reply:
I've deleted ListEvent in this PR, which would cause tests to fail, without updating ScheduledEvent